### ISSUE TITLE
Duplicate argument definition throws exception, but that exception is not given to the user.

### ISSUE THREAD

**Issue by LeeTL1220** [broadinstitute/barclay#27]
```
lichtens@OncobuntuMk3:~/IdeaProjects/hellbender-protected$ java -jar build/libs/gatk-protected.jar GetHetCoverageLocusWalker
lichtens@OncobuntuMk3:~/IdeaProjects/hellbender-protected$ 
```

**Comment by cmnbroad** [comment]
This will be fixed by https://github.com/broadinstitute/gatk/pull/2340 (Barclay  https://github.com/broadinstitute/barclay/pull/24 would probably also fix it, but that requires a Barclay upgrade) and https://github.com/broadinstitute/gatk/pull/2340 as it currently stands would be sufficient for the case of duplicate fields.

**Comment by cmnbroad** [comment]
Closed via #24 and broadinstitute/gatk#2340.
